lib/chef/guard_interpreter/default_guard_interpreter.rb

```diff
@@ -33,7 +33,9 @@ def initialize(command, opts)
       public
 
       def evaluate
-        shell_out(@command, @command_opts).status.success?
+        result = shell_out(@command, @command_opts)
+        Chef::Log.warn "Command failed: #{result.stderr}" unless result.status.success?
```

Comment:
oh good point re: failure being a normal (or at least expected) condition... i'll update this to debug.